packages/wrangler/src/__tests__/helpers/msw/index.ts

```diff
@@ -7,8 +8,34 @@ import { mswSuccessR2handlers } from "./handlers/r2";
 import { default as mswSucessScriptHandlers } from "./handlers/script";
 import { mswSuccessUserHandlers } from "./handlers/user";
 import { default as mswZoneHandlers } from "./handlers/zones";
+import type { MockedRequest } from "msw";
 export const msw = setupServer();
 
+function waitForRequest(method: string, url: string) {
```

Comment:
Oh, maybe I can use this for the `login` mocking stuff, that hits the browser for OAuth